cmd/swaptester/main.go

```diff
@@ -206,7 +206,7 @@ var (
 )
 
 func getRandomExchangeRate() *coins.ExchangeRate {
-	rateFl := minExchangeRate + mrand.Float64()*(maxExchangeRate-minExchangeRate) //nolint:gosec
+	rateFl := minExchangeRate + mrand.Float64()*(maxExchangeRate-minExchangeRate)
```

Comment:
Don't know why these changed. Maybe because it is a float value, which wouldn't normally be using for cryptography.